bin/node/pallets/pallet-updater/src/lib.rs

```diff
@@ -106,7 +156,73 @@ pub mod pallet {
 			// https://docs.substrate.io/rustdocs/latest/sp_std/vec/struct.Vec.html#method.retain
 			<Members<T>>::mutate(|v| v.retain(|x| x != &remove_acct));
 
-			Self::deposit_event(Event::RemovedUpdaters(remove_acct));
+			Self::deposit_event(Event::RemovedUpdater(remove_acct));
+
+			Ok(())
+		}
+
+		/// Adds a proposal to be voted on.
+		///
+		/// Requires the sender to be a member.
+		#[pallet::weight(10_000 + T::DbWeight::get().writes(1))]
+		pub fn propose_code(origin: OriginFor<T>, code: Vec<u8>) -> DispatchResult {
```

Comment:
What is this? What does `code: Vec<u8>` means in our context?

Reply:
> What is this? What does `code: Vec<u8>` means in our context?

I thought that we could give as input the code of the upgrade (like in the set_code function).